### PR TITLE
Add workdir input

### PR DESCRIPTION
**Pull request by timharris777**
I have projects where I am checking out multiple repos to different directories. To support running this action on a git repo that was in a different directory I had to add the workdir input option. 

- All steps in the [CONTRIBUTING guidelines](https://github.com/crazy-max/ghaction-import-gpg/blob/master/.github/CONTRIBUTING.md) were followed.
- All tests were passing.

### CODE REVIEW

**Review by crazy-max — CHANGES_REQUESTED** [review]
Please update the documentation ([inputs section in the README](https://github.com/crazy-max/ghaction-import-gpg#inputs)).

**Comment by timharris777** [comment]
The readme has been updated

File: README.md [hunk 1]
```diff
@@ -137,6 +139,7 @@ Following inputs can be used as `step.with` keys
 | `git_push_gpgsign`**¹**               | Bool    | Sign all pushes automatically. (default `false`) |
 | `git_committer_name`**¹**             | String  | Set commit author's name (defaults to the name associated with the GPG key) |
 | `git_committer_email`**¹**            | String  | Set commit author's email (defaults to the email address associated with the GPG key) |
+| `workdir`**¹**                        | String  | Working directory (below repository root) |
```
Comment (crazy-max) [inline README.md:142]: Remove `**¹**`

File: README.md [hunk 2]
```diff
@@ -15,17 +15,19 @@ If you are interested, [check out](https://git.io/Je09Y) my other :octocat: GitH
...
+- [Usage](#usage)
+  - [Workflow](#workflow)
+  - [Sign commits](#sign-commits)
+- [Customizing](#customizing)
+  - [inputs](#inputs)
+  - [outputs](#outputs)
+  - [environment variables](#environment-variables)
+- [Keep up-to-date with GitHub Dependabot](#keep-up-to-date-with-github-dependabot)
+- [How can I help?](#how-can-i-help)
+- [License](#license)
```
Comment (crazy-max) [inline README.md:30]: I prefer to keep `* ` for unordered lists. Please undo these changes.

**Review by crazy-max — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (timharris777) [reply README.md:30]: Sorry it was my toc plugin. Undoing them now...

Comment (timharris777) [reply README.md]: How is that?

File: README.md (hunk 1 above)
Comment (timharris777) [reply README.md:141]: Taking a look at how that got added.

Comment (timharris777) [reply README.md]: I copied the line above. Dohhh... It's fixed now.

**Review by crazy-max — APPROVED** [review]
LGTM thanks!
